community/kernel/src/main/java/org/neo4j/kernel/impl/recovery/StoreRecoveryChecker.java

```diff
@@ -36,24 +33,20 @@
 import org.neo4j.kernel.impl.transaction.log.entry.LogEntryReaderFactory;
 import org.neo4j.kernel.recovery.LatestCheckPointFinder;
 import org.neo4j.kernel.recovery.PositionToRecoverFrom;
-import org.neo4j.logging.LogProvider;
 
 /**
- * For now, an external tool that can determine if a given store will need
- * recovery, and perform recovery on given stores.
+ * An external tool that can determine if a given store will need recovery.
  */
-public class StoreRecoverer
+public class StoreRecoveryChecker
```

Comment:
Hm, I wonder if _StoreStateChecker_ would be a better name. _StoreRecoveryChecker_ might also be the name of a class that checks if a recovery has been successful.